builtin_error.go

```diff
@@ -124,6 +124,7 @@ func (r *Runtime) builtin_Error(args []Value, proto *Object) *Object {
 	if len(args) > 0 && args[0] != _undefined {
 		obj._putProp("message", args[0], true, false, true)
 	}
+	// todo might not be needed? It isn't in the upstream
```

Comment:
Probably best to just leave alone, it might not exist due to other changes we haven't pulled in

Reply:
I will leave alone 👍